src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -308,8 +466,28 @@
                 </Grid>
             </Border>
 
+            <ComboBox Name="FlowsPicker" 
+                      Grid.Row="1" 
+                      Style="{StaticResource FlowsComboBox}"
+                      ItemContainerStyle="{StaticResource FlowsComboBoxItemStyle}"
+                      ItemsSource="{Binding CurrentIssue.Flows}"
+                      SelectedItem="{Binding CurrentFlow, Mode=TwoWay}">
+                <ComboBox.ItemTemplate>
+                    <DataTemplate DataType="{x:Type models:IAnalysisIssueFlowVisualization}">
+                        <TextBlock Style="{StaticResource BaseNumberInShapeTextBlockStyle}">
+                            <TextBlock.Text>
+                                <MultiBinding StringFormat="{}Flow {0} ({1} locations)">
```

Comment:
What is the empty `{}` for?

Reply:
Leftovers from initial attempts - you need to add {} if there is no custom text, and it first it was just flow number.